### ISSUE TITLE
Bug: Unable to create connection with DuckDB data source in Windows environment.

### ISSUE THREAD

**Issue by keithbritt** [ibis-project/ibis#6038]
### What happened?

I've tried this also with a separate data source that was a csv imported and saved as a ddb file, but I double checked with the getting started dataset and get the same exact result.

Here's the code I'm running:

import ibis
import urllib.request
urllib.request.urlretrieve(
        "https://storage.googleapis.com/ibis-tutorial-data/palmer_penguins.ddb",
        "palmer_penguins.ddb",
    )
ibis.options.interactive = True
ibis.options.verbose     = True

con = ibis.duckdb.connect("palmer_penguins.ddb")
con.list_tables()

birds = con.table("penguins")

And here's the command line I run along with the error message I received (it doesn't just warn, it fails and terminates). Notice it's a Postgres error even though I'm using DuckDB.

py Hello_World.py

...Python\Python310\site-packages\ibis\backends\postgres\registry.py:119: UserWarning: locale specific date formats (%c, %x, %X) are not yet implemented for Windows
  warnings.warn(

### What version of ibis are you using?

5.0

### What backend(s) are you using, if any?

DuckDB

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cpcloud** [comment]
@keithbritt Thanks for the issue!

Can you show the full error message? You're showing only the warning, but then you say that it's not only warning but a failure and termination.

Can you show exactly what happens?

**Comment by keithbritt** [comment]
That's all the information I'm getting. Here's a screen shot if that's helpful.

<img width="1082" alt="image" src="https://user-images.githubusercontent.com/7809612/233086794-b7207473-8fa0-47a9-b403-5f09482f1aa8.png">

**Comment by cpcloud** [comment]
I don't think that's a failure. Your program appears to be terminating normally.

**Comment by cpcloud** [comment]
This code: ```
import ibis
import urllib.request
urllib.request.urlretrieve(
"[storage.googleapis.com/ibis-tutorial-data/palmer_penguins.ddb](https://storage.googleapis.com/ibis-tutorial-data/palmer_penguins.ddb)",
"palmer_penguins.ddb",
)
ibis.options.interactive = True
ibis.options.verbose = True

con = ibis.duckdb.connect("palmer_penguins.ddb")
con.list_tables()

birds = con.table("penguins")
```

will not show any output.

You need to take an action that prints to stdout.

For example, you could add

```
print(birds)
```

as the final line of code.

**Comment by keithbritt** [comment]
You're right. I misinterpret the warning. The connection does succeed and I am getting back data. Still though, I guess where I got the mental block was seeing the Postgres warning pop up for a DuckDB connection. Could potentially hang up some other users. Thanks for your help.

**Comment by keithbritt** [comment]
And just for the record... it is formatting the DuckDB date data correctly, despite the date format warning for Postgres.

**Comment by cpcloud** [comment]
@keithbritt Yep, we can probably make that warning more precise!

The warning is specific to windows *and* to the contents of the format string.

Thanks bringing this up, and happy to answer questions.
